lisa/testsuite.py

```diff
@@ -925,6 +927,14 @@ def _add_case_to_suite(
 ) -> None:
     test_case.suite = test_suite
     test_case.full_name = f"{test_suite.name}.{test_case.name}"
+    # Append tags from test_suite if exists to test_case tags and remove duplicates
+    suite_tags = getattr(test_suite, "tags", None)
+    case_tags = getattr(test_case, "tags", None)
+    if suite_tags:
+        if case_tags is None:
+            test_case.tags = list(dict.fromkeys(suite_tags))
+        else:
+            test_case.tags = list(dict.fromkeys(case_tags + suite_tags))
```

Comment:
Try below, it's simpler.
```suggestion
    suite_tags = getattr(test_suite, "tags", [])
    case_tags = getattr(test_case, "tags", [])
    test_case.tags = list(dict.fromkeys(case_tags + suite_tags))
```

Reply:
Sure, updated